ArduPlane/ArduPlane.cpp

```diff
@@ -505,6 +505,11 @@ void Plane::update_fly_forward(void)
         return;
     }
 
+    if (quadplane.in_transition()) {
+        ahrs.set_fly_forward(true);
+        return;
+    }
+
     if (quadplane.in_vtol_mode() ||
         quadplane.in_assisted_flight()) {
```

Comment:
Couldn't removing this line amount to the same thing?

Reply:
No longer relevant, I think - the option bit makes it different